src/main/java/com/fasterxml/jackson/databind/deser/std/ThrowableDeserializer.java

```diff
@@ -120,6 +122,14 @@ public Object deserializeFromObject(JsonParser p, DeserializationContext ctxt) t
                     continue;
                 }
             }
+
+            // Maybe it's "suppressed"?
+            final boolean isSuppressed = PROP_NAME_SUPPRESSED.equals(propName);
+            if (isSuppressed) {
+                suppressed = p.readValueAs(Throwable[].class);
```

Comment:
Instead of calling via `JsonParser`, should go through `DeserializationContext`; this will keep current context and settings (`JsonParser` route will create new context with possibly differing settings -- at least until Jackson 3.0)

Reply:
OK, changed the corresponding line to

```
suppressed = ctxt.readValue(p, Throwable[].class);
```